src/deployment/android-release.md

```diff
@@ -2,8 +2,9 @@
 title: Preparing an Android App for Release
 ---
 
-During the typical development cycle, you'll test an app using `flutter run` at the
-command line, the Run and Debug toolbar buttons in IntelliJ, or both. By default,
+During the typical development cycle, you'll test an app using
+`flutter run` at the command line, the Run and Debug toolbar buttons
```

Comment:
the Run and Debug toolbar buttons -> the **Run** and **Debug** toolbar buttons (per guidelines)